src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html

```diff
@@ -263,73 +261,65 @@
     <!-- Encryption -->
     <fieldset>
       <div class="form-item">
-        <legend class="cd-header"
-                i18n>Encryption</legend>
-        <cds-checkbox name="encryption_enabled"
-                      formControlName="encryption_enabled"
-                      id="encryption_enabled"
-                      [disabled]="!kmsConfigured && !s3Configured"
-                      i18n>Enable
+        <legend class="cd-header" i18n>Encryption</legend>
+        <cds-checkbox name="encryption_enabled" formControlName="encryption_enabled" id="encryption_enabled" i18n>Enable
           <cd-help-text aria-label="encryption helper">
             <span>Enables encryption for the objects in the bucket.
-                To enable encryption on a bucket you need to set the configuration values for SSE-S3 or SSE-KMS.
-                To set the configuration values <a href="#/rgw/configuration"
-                                                   i18n-aria-label="click here">Click here</a></span>
+              To enable encryption on a bucket you need to set the configuration values for SSE-S3 or SSE-KMS.
+              To set the configuration values <a href="#/rgw/configuration" i18n-aria-label="click here">Click
+                here</a></span>
           </cd-help-text>
         </cds-checkbox>
       </div>
-
+    
       <div *ngIf="bucketForm.getValue('encryption_enabled')">
         <div class="form-item">
           <cds-radio-group formControlName="encryption_type">
-          <cds-radio value="AES256"
-                     [disabled]="!s3Configured">SSE-S3</cds-radio>
-          <cds-radio value="aws:kms"
-                     [disabled]="!kmsConfigured">Connect to an external key management service</cds-radio>
+            <cds-radio value="AES256" [ngClass]="{'disabled-radio': !isS3RadioEnabled}"
+              (click)="!isS3RadioEnabled && $event.preventDefault()">SSE-S3
+              <cds-tooltip *ngIf="!isS3RadioEnabled"
+                [description]="'SSE-S3 is not configured. Please configure it in the RGW settings.'" [highContrast]="true"
+                [caret]="true">
+                <svg cdsIcon="info" size="16" class="cds--radio__info"></svg>
+              </cds-tooltip>
+            </cds-radio>
+            <cds-radio value="aws:kms" [ngClass]="{'disabled-radio': !isKmsRadioEnabled}"
+              (click)="!isKmsRadioEnabled && $event.preventDefault()">Connect to an external key management service
+              <cds-tooltip *ngIf="!isKmsRadioEnabled"
+                [description]="'SSE-KMS is not configured. Please configure it in the RGW settings.'" [highContrast]="true"
+                [caret]="true">
+                <svg cdsIcon="info" size="16" class="cds--radio__info"></svg>
+              </cds-tooltip>
+            </cds-radio>
           </cds-radio-group>
```

Comment:
I noticed that the warning is coming for only checkboxes and not radios so we dont need to fix radios.